sql/src/main/java/io/crate/protocols/postgres/types/PGType.java

```diff
@@ -86,7 +90,15 @@ public final int writeAsText(ChannelBuffer buffer, @Nonnull Object value) {
     public Object readTextValue(ChannelBuffer buffer, int valueLength) {
         byte[] bytes = new byte[valueLength];
         buffer.readBytes(bytes);
-        return decodeUTF8Text(bytes);
+        try {
+            return decodeUTF8Text(bytes);
+        } catch (Throwable t) {
+            if (LOGGER.isWarnEnabled()) {
```

Comment:
I call warn with `new String...` which would be evaluated even if warn is disabled. This check here is to prevent that.